app/javascript/components/ansible-repository-form/helpers.jsx

```diff
@@ -0,0 +1,16 @@
+
+// NOTE: ddfUrlRegex is the regex output of https://github.com/data-driven-forms/react-forms/blob/master/packages/react-form-renderer/src/validators/url-validator.js#L60-L80 this file 
```

Comment:
As time moves on, these lines of code could move, but if you pin to a SHA, it won't

```suggestion
// NOTE: ddfUrlRegex is the regex output of https://github.com/data-driven-forms/react-forms/blob/3199d495361313616901f52ba2da6a21cb3d3a50/packages/react-form-renderer/src/validators/url-validator.js#L60-L80 this file 
```